label_studio/tests/tasks/test_presign_storage_data.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from unittest.mock import patch
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIRequestFactory, force_authenticate
+from unittest.mock import MagicMock
+from data_import.api import PresignStorageData
+from users.models import User
+from tasks.models import Task
+from projects.models import Project
+
+
+@pytest.mark.django_db
```

Comment:
it's better to write integration tests in tavern: 
https://www.notion.so/heartexheartex/Backend-Handbook-ce7324a497cb4b40b91911a429e34fcb?pvs=4#f1f17dba772743d3b628b1dc7bd7def9

Reply:
Done #4091 and left these tests as well, since the coverage was taking a large hit without these.